.github/workflows/update-metainfo.yml

```diff
@@ -0,0 +1,21 @@
+name: Update Metainfo
+
+on:
+  release:
+    types: [released]
```

Comment:
Does the XML update have to be in the release as well or can it just be in the latest commit outside the release?

Reply:
It can be in the latest commit outside the release. I fetch that file manually in manifest, so the old one is always replaced with the updated file.